components/server/ee/src/gitlab/gitlab-app-support.ts

```diff
@@ -61,4 +61,18 @@ export class GitLabAppSupport {
         result.unshift(...ownersRepos);
         return result;
     }
+
+    protected async getAccountAvatarUrl(anyProject: any, providerHost: string): Promise<string> {
+        if (anyProject.owner?.avatar_url) {
+            return anyProject.owner.avatar_url;
+        }
+        if (anyProject.namespace?.avatar_url) {
+            let url = anyProject.namespace.avatar_url;
```

Comment:
**issue**: Not sure if this is the correct line to comment this but the avatar seems broken now while before it was just a white blank space, see screenshots below.

| BEFORE |  AFTER|
|-|-|
| <img width="346" alt="list-before" src="https://user-images.githubusercontent.com/120486/158410173-fd1b2e71-6f31-40b0-b2af-22dc7c8577af.png"> | <img width="346" alt="list-after" src="https://user-images.githubusercontent.com/120486/158410180-58699953-f535-4934-9cd7-6a7a356c0fc9.png"> |

Reply:
Ah, thanks for catching this @gtsiolis. If you inspect the broken icons, what does the avatar URL look like?

Reply:
**praise**: Thanks for fixing this, work like a charm. ✨

Reply:
The groups with custom avatars still seems broken. What do you think of opening a follow up issue for this to keep track of this?

<img width="489" alt="Frame 370" src="https://user-images.githubusercontent.com/120486/161529256-f313f045-110a-47ff-afba-1bb4647e8737.png">

Reply:
It seems to be working ok now for all but one of the groups in there, posted a comment in https://github.com/gitpod-io/gitpod/issues/9112#issuecomment-1087787341.